### PR TITLE
Fix #91571: Add toolbar toggles to View menu

### PR DESCRIPTION
**Pull request by IsaacWeiss**

### CODE REVIEW

File: mscore/shortcut.cpp [hunk 1]
```diff
@@ -1769,20 +1769,41 @@ Shortcut Shortcut::_sc[] = {
...
+         "toggle-imagecapture",
+         QT_TRANSLATE_NOOP("action","Image Capture"),
+         QT_TRANSLATE_NOOP("action","Image Capture toolbar")
+         },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT ,
+         "toggle-noteinput",
+         QT_TRANSLATE_NOOP("action","Note Input"),
+         QT_TRANSLATE_NOOP("action","Note Input toolbar")
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `input`, lowercase

File: mscore/shortcut.cpp [hunk 2]
```diff
@@ -1769,20 +1769,41 @@ Shortcut Shortcut::_sc[] = {
...
+         "toggle-concertpitch",
+         QT_TRANSLATE_NOOP("action","Concert Pitch"),
+         QT_TRANSLATE_NOOP("action","Concert Pitch toolbar")
+         },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT ,
+         "toggle-imagecapture",
+         QT_TRANSLATE_NOOP("action","Image Capture"),
+         QT_TRANSLATE_NOOP("action","Image Capture toolbar")
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `capture`, lowercase

File: mscore/shortcut.cpp [hunk 3]
```diff
@@ -1769,20 +1769,41 @@ Shortcut Shortcut::_sc[] = {
...
+         "toggle-transport",
+         QT_TRANSLATE_NOOP("action","Playback Controls"),
+         QT_TRANSLATE_NOOP("action","Playback Controls toolbar")
+         },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT ,
+         "toggle-concertpitch",
+         QT_TRANSLATE_NOOP("action","Concert Pitch"),
+         QT_TRANSLATE_NOOP("action","Concert Pitch toolbar")
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `pitch`, lowercase

File: mscore/shortcut.cpp [hunk 4]
```diff
@@ -1769,20 +1769,41 @@ Shortcut Shortcut::_sc[] = {
...
+         "toggle-fileoperations",
+         QT_TRANSLATE_NOOP("action","File Operations"),
+         QT_TRANSLATE_NOOP("action","File Operations toolbar")
+         },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT ,
+         "toggle-transport",
+         QT_TRANSLATE_NOOP("action","Playback Controls"),
+         QT_TRANSLATE_NOOP("action","Playback Controls toolbar")
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `controls', lowercase (much better that then previous `Transport` BTW)

File: mscore/shortcut.cpp [hunk 5]
```diff
@@ -1769,20 +1769,41 @@ Shortcut Shortcut::_sc[] = {
...
-//         "toggle-noteinput",
-//         QT_TRANSLATE_NOOP("action","Note Input"),
-//         QT_TRANSLATE_NOOP("action","Note input toolbar")
-//         },
+      {
+         MsWidget::MAIN_WINDOW,
+         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT ,
+         "toggle-fileoperations",
+         QT_TRANSLATE_NOOP("action","File Operations"),
+         QT_TRANSLATE_NOOP("action","File Operations toolbar")
```
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `operations`, lowercase

File: mscore/shortcut.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: `input`, lowercase, just like it was before

Comment (IsaacWeiss) [inline mscore/shortcut.cpp]: I thought of this as referencing the [title of the toolbar](https://github.com/musescore/MuseScore/pull/2923/files#diff-3056168911c88e142262cb695efd9fdaR1370), as if in quotes: <code>"Note Input" toolbar</code> But I'm not opposed to changing it.

File: mscore/shortcut.cpp (hunk 4 above)
Comment (IsaacWeiss) [inline mscore/shortcut.cpp]: Thanks.

File: mscore/shortcut.cpp (hunk 1 above)
Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: I'm just looking at how all the other are done.

Comment (IsaacWeiss) [inline mscore/shortcut.cpp]: Other whats?

Comment (Jojo-Schmitz) [inline mscore/shortcut.cpp]: other strings in that Position. They are all Sentence case. The ones above all in Title Case

Comment (IsaacWeiss) [inline mscore/shortcut.cpp]: Sure, OK.

Comment (IsaacWeiss) [inline mscore/shortcut.cpp]: All made lowercase.

**Comment by Jojo-Schmitz** [comment]
I guess we'd want this for 2.1 too, right?

**Comment by IsaacWeiss** [comment]
I guess so. Separate PR needed for that.

**Pull request merged**